ospf6d/ospf6_network.c

```diff
@@ -119,11 +129,14 @@ int ospf6_sso(ifindex_t ifindex, struct in6_addr *group, int option)
 	int ret;
 	int bufsize = (8 * 1024 * 1024);
 
+	if (ospf6->fd < 0)
```

Comment:
Nit: I think the more correct check is `if (ospf6->fd == -1)`.

Reply:
Fixed